blueprints/addon/index.js

```diff
@@ -77,6 +82,19 @@ module.exports = {
     // 100% of addons don't need ember-cli-app-version, make it opt-in instead
     delete contents.devDependencies['ember-cli-app-version'];
 
+    // add scripts to build type declarations for TypeScript addons
+    if (this.options.typescript) {
+      contents.scripts.prepack = 'tsc --project tsconfig.declarations.json';
+      contents.scripts.postpack = 'rimraf declarations';
```

Comment:
The `rimraf` command is not available by default when running `yarn pack`. Not sure if that's considered an issue?

Reply:
Should be fixed now!